package sc14_tratamento_excecoes.ex03_excecoes_personalizadas.model.exceptions;

public class DomainException extends Exception {
    private static final long serialVersionUID = 1L;

    public DomainException(String msg) {
        super(msg);
    }
    
}
